docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -119,7 +119,8 @@ After the shebang, the UTF-8 coding, the copyright line, the license, and the ``
 Module documentation should briefly and accurately define what each module and option does, and how it works with others in the underlying system. Documentation should be written for broad audience--readable both by experts and non-experts.
     * Descriptions should always start with a capital letter and end with a full stop. Consistency always helps.
     * Verify that arguments in doc and module spec dict are identical.
-    * For password / secret arguments no_log=True should be set.
+    * For password / secret arguments ``no_log=True`` should be set.
+    * For arguments that are redacted automatically (e.g. "password_length"), but are not password / secret arguments, set ``no_log=False``.
```

Comment:
```suggestion
    * For arguments that seem to contain sensitive information but **do not** contain secrets, such as "password_length", set ``no_log=False`` to disable the warning message.
```

Reply:
We avoid using Latin abbreviations like 'e.g.' that are less familiar to those for whom English is not their native language.